perf/benchmark/run_benchmark_job.sh

```diff
@@ -32,13 +32,14 @@ export FORTIO_CLIENT_URL=""
 export LC_ALL=C.UTF-8
 export LANG=C.UTF-8
 export GCS_BUCKET="istio-build/perf"
+export NAMESPACE="${NAMESPACE:-'twopods'}"
```

Comment:
```suggestion
export NAMESPACE="${NAMESPACE:-twopods}"
```

Reply:
@jwendell Could you please take another look? Thank you

Reply:
@carolynhu Done. It now has some conflicts.